ReactWindows/ReactNative/Touch/TouchHandler.cs

```diff
@@ -97,7 +97,16 @@ private void OnPointerConcluded(TouchEventType touchEventType, PointerRoutedEven
                 UpdatePointerForEvent(pointer, e);
                 DispatchTouchEvent(touchEventType, _pointers, pointerIndex);
 
-                --_activePointers;
+                if (_activePointers == pointerIndex + 1)
+                {
```

Comment:
Can you explain what's going on here?  Is this a bug in XAML that's losing track of pointers?